.gitignore

```diff
@@ -42,3 +42,4 @@ node_modules
 
 /bazel-*
 CMakeLists.txt.user
+CMakeFiles
```

Comment:
This looks fine to me, but you can also specify a sub-directory for cmake to place artifacts. ex: `cmake . -B build -G Xcode` to place CMakeFiles, generated Xcode project and all other artifacts in /build (which is already under the gitignore).

Reply:
Ah, right, those are probably just left over from messing around with CMake previously.